### PR TITLE
shipit_code_coverage: Load bugs summary, refs #319

### PR DESCRIPTION
**Pull request by La0**
And display on shipit_frontend

### CODE REVIEW

**Comment by La0** [comment]
For reference, this is a [screenshot](http://i.imgur.com/uugD8wn.png) of the display with bugs summary

File: src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py [hunk 1]
```diff
@@ -95,6 +104,16 @@ def generate(rootDir):
...
+        bugs.update([b['bugzilla_id'] for b in data[directory]['bugs']])
+
+    def _bughandler(bug, *args, **kwargs):
+        for directory in data.values():
+            bug_id = bug['id']
+            for b in directory['bugs']:
+                if b['bugzilla_id'] == bug_id:
+                    b['summary'] = bug['summary']
+
+    Bugzilla(bugs, bughandler=_bughandler).get_data().wait()
```
Comment (marco-c) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: Let's use `include_fields` to make the query lighter for bugzilla

File: src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py [hunk 2]
```diff
@@ -41,7 +42,14 @@ def load_changesets(rev1, rev2):
 
 
 def get_related_bugs(changesets, directory):
-    return list(set([bug for bug, paths in changesets if any(fnmatch.fnmatch(path, directory + '/*') for path in paths)]))
+    bugs = set([bug for bug, paths in changesets if any(fnmatch.fnmatch(path, directory + '/*') for path in paths)])
```
Comment (marco-c) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: Nit: Can you make this return the list and then in `generate` create the dict for each bug?

Comment (La0) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: Sure

File: src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py (hunk 1 above)
Comment (La0) [inline src/shipit_code_coverage/shipit_code_coverage/coverage_by_dir.py]: :+1: I knew there was some way to do that, thanks !

**Pull request merged**